Client/mods/deathmatch/logic/luadefs/CLuaGUIDefs.cpp

```diff
@@ -795,11 +795,13 @@ int CLuaGUIDefs::GUICreateStaticImage(lua_State* luaVM)
             if (CResourceManager::ParseResourcePathInput(path, pResource, &strPath))
             {
                 CClientGUIElement* pGUIElement = CStaticFunctionDefinitions::GUICreateStaticImage(*pLuaMain, position, size, strPath, relative, parent);
-                lua_pushelement(luaVM, pGUIElement);
-                return 1;
+                if (pGUIElement != NULL)
```

Comment:
IIRC, we now try to use `nullptr` instead of `NULL`

Reply:
i don't know which variant i should use

Reply:
never use `NULL`, always use `nullptr`

Reply:
done